Control.cpp

```diff
@@ -113,4 +76,18 @@ double Control::getNewDesireRight(double new_desire) {
   }
   else{}
   return new_desire;
+}
+
+bool Control::detectRamp() {
+  if (logic.getAngleBNOY() < -4 && logic.getAngleBNOY() > -35) {
+    if (logic.getAngleBNOZ() < 3 && logic.getAngleBNOZ() > -3) {
+      return true;
+    }
+    else{
+      return false;
+    }
+  }
+  else{
+    return false;
+  }
```

Comment:
remember how you could simplify this?
Also, the -4, -35, 3 and -3 make them constants.

Reply:
Hint: Which conditions are TRUE when you return TRUE